standards/HEALTH_CHECK_SPECIFICATION.md

```diff
@@ -0,0 +1,101 @@
+Health check specification
+==========================
+
+All apps that run in production are required to have an http health check endpoint, even if they are event driven.  This health check must comply with [the spec](#health-check-spec). The main function of these health checks is to enable the apps to signal to the platform whether or not they are functional.  The platform will then use this information to manage routing to the app and the lifecycle of the app.
+
+The health check can be looked at in two parts:
+
+1. [The checks](#checks)
+2. [The health check endpoint](#health-check-endpoint)
+
+Checks
+------
+
+The purpose of a health check is so that a monitoring tool can quickly determine the current fitness of that app to perform its function.  This means that a health check can have many different checks, and indeed many different types of checks.  These checks can be against external dependencies or against the internal functioning of the app.
```

Comment:
```suggestion
The purpose of a health check is so that a monitoring tool can quickly determine the current fitness of that app to perform its function.  This means that a health check can have many different checks, and indeed many different types of checks.  These checks can be against external dependencies or the internal functioning of the app.
```
The preposition 'against' is redundant as it is stated on the first condition of the 'or' in the statement.